src/main/java/core/basesyntax/servise/fileservice/FileReportWriter.java

```diff
@@ -0,0 +1,5 @@
+package core.basesyntax.servise.fileservice;
+
+public interface FileReportWriter {
+    String writeReportToFile(String report);
```

Comment:
1. I don't think writer contain some specific logic for report so better to use more general naming that comply with reader
2. Please pass path as argument, don't hardcode in service
```suggestion
public interface FileWriterUtility {
    String writeReportToFile(String content, String fileName);
```